src/routes/safe/store/actions/createTransaction.js

```diff
@@ -104,17 +109,17 @@ const createTransaction = ({
   try {
     if (isExecution) {
       tx = await getExecutionTransaction(
-        safeInstance, to, valueInWei, txData, CALL, nonce,
+        safeInstance, to, valueInWei, txData, operation, nonce,
         0, 0, 0, ZERO_ADDRESS, ZERO_ADDRESS, from, sigs,
       )
     } else {
       tx = await getApprovalTransaction(
-        safeInstance, to, valueInWei, txData, CALL, nonce,
+        safeInstance, to, valueInWei, txData, operation, nonce,
         0, 0, 0, ZERO_ADDRESS, ZERO_ADDRESS, from, sigs,
       )
     }
 
-    const sendParams = { from, value: 0 }
+    const sendParams = { from, value: 0, gasLimit: 1000000 }
```

Comment:
Can we add a "TODO" or something similar to remove this again, we should not hardcode a gaslimit here and this should be removed before merging

Reply:
sure

Reply:
@nicosampler can you coordinate with @Agupane?

he based some changes in this code, and there'll be conflicts in the near future (#544)

Reply:
> Can we add a "TODO" or something similar to remove this again, we should not hardcode a gaslimit here and this should be removed before merging
> 
> Edit: Maybe only set this for certain envs (e.g. rinkeby local)

@rmeissner  what do you think about `REACT_APP_ENV !== "production"`?

Reply:
done!